### ISSUE TITLE
This repository is only an empty shell

### ISSUE THREAD

**Issue by kleberbaum** [aichner/Wagtail-Template#4]
**Is your feature request related to a problem? Please describe.**
Currently this repository is only an empty shell.

**Describe the solution you'd like**
I would like to start a project by the name esite w/ two default apps.

Typically I name all my django projects esite as a homage to my old website.
"Sky's The Limit" - boiz xD

The structure should be:
```
//> django project
- esite
  + core  // This app should contain all modifications on the core.
  + home  // This app should always contain the homepage.
```

**Comment by kleberbaum** [comment]
This feature request was granted and has been implemented in pull request #6.

**Comment by kleberbaum** [comment]
This feature request was granted and has been implemented in pull request #6.
